sdk/netapp/arm-netapp/samples-dev/netAppResourceCheckFilePathAvailabilitySample.ts

```diff
@@ -5,20 +5,26 @@
  * Code generated by Microsoft (R) AutoRest Code Generator.
  * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
-import type { NetAppResourceCheckFilePathAvailabilityOptionalParams } from "@azure/arm-netapp";
-import { NetAppManagementClient } from "@azure/arm-netapp";
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
I remember we have removed the duplicated license header in codegen?

Reply:
We don't seem to have done this